shark/shark_importer.py

```diff
@@ -0,0 +1,112 @@
+# Lint as: python3
+"""SHARK Importer"""
+
+import iree.compiler.tflite as iree_tflite_compile
+import iree.runtime as iree_rt
+import numpy as np
+import os
+import sys
+import tensorflow.compat.v2 as tf
+import time
+import urllib.request
+from shark.shark_inference import SharkInference
+
+targets = {
+  'dylib' : 'dylib-llvm-aot',
+  'vulkan' : 'vulkan-spirv',
+  'cuda' : 'cuda',
+}
+
+class GenerateInputSharkImporter():
+  def __init__(self, input_details, model_source_hub="tfhub"):
+    self.input_details = input_details
+    self.model_source_hub = model_source_hub
+
+  def generate_inputs(self):
+    args = []
```

Comment:
Seems like AlbertInput can be stand alone without this base class, since we override the generate input anyways. and all we use from this class becomes just the setting of input_details and model_source_hub

Reply:
done